### PR TITLE
Add Titan model support

### PR DESCRIPTION
**Pull request by pacphi**
It's a beautiful day in the Gen AI neighborhood!

### CODE REVIEW

**Review by honnuanand — COMMENTED** [review]
@pacphi, my initial idea was "aws.bedrock.model.id" to define what model needs to activated, instead of "aws.bedrock.model.titan.id" what if I activate "aws.bedrock.model.claude.model.id" as well? That's why I thought, we could have the model id at a level higher. Please let me know what you think.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,14 @@ aws.bedrock.model.llama2.maxTokens=200
 aws.bedrock.model.llama2.temperature=0.5
 ```
 
+#### Titan model parameters
+
+```properties
+aws.bedrock.model.id=amazon.titan-text-express-v1
```
Comment (honnuanand) [inline README.md:78]: This is the right level,  then what is "aws.bedrock.model.titan.id" for .. it gets a little confusing. Let's chat about what the props should look like for each model. I would prefer not to activate more than one model at a time unless you convince me it is a good idea.

Comment (pacphi) [reply README.md:78]: This is a mistake.  Should be aws.bedrock.model.titan.id. I'll push an update to the docs.  Behind the scenes, this is the case.

Comment (pacphi) [reply README.md:78]: I think we shouldn't limit the number of services activated.  Consumers could specify @Primary and @Qualifier annotations to override auto-config.

**Review by honnuanand — APPROVED** [review]

**Pull request merged**